src/common/components/signInForm/signInForm.component.js

```diff
@@ -51,6 +51,14 @@ class SignInFormController {
               this.signInState = 'identity'
               this.onFailure()
           }
+        } else if (error && error.data && error.data.code && error.data.code === 'SIEB-DOWN') {
```

Comment:
Does the tooling support this? 
```suggestion
        } else if (error?.data?.code === 'SIEB-DOWN') {
```
You can merge as is since it's probably not worth the time testing it.

Reply:
Hmm, not sure, there isn't any typescript in this project as far as I know.